deployments/container/cleanup/cleanup.sh

```diff
@@ -52,6 +52,10 @@ microdnf remove -y \
 microdnf remove -y \
     $(rpm -qa | sort | grep -v -f package-names.minimal -e gpg-pubkey)
 
+# We need mount for our init container and install it here.
+# TODO: We need to provide a better way to install this / skip the cleanup of this package.
+microdnf install --assumeyes util-linux-core
```

Comment:
Non-blocking. Only because `--assumeyes` is such an eyesore
```suggestion
microdnf install -y util-linux-core
```

Reply:
Addressed in #1013